addon/models/sign.ts

```diff
@@ -4,6 +4,7 @@ import { unwrap } from '../utils/option';
 export default class Sign {
   constructor(
     readonly code: string,
+    readonly signType: string,
```

Comment:
You can rename this type as it's already inside the sign :)

Reply:
that's ok for now